src/app/rosetta/lib/account.ml

```diff
@@ -131,6 +133,18 @@ module Balance = struct
         | Some state_hash ->
             M.return state_hash
       in
+      let%map liquid_balance =
+        match (account#balance)#liquid with
+        | None ->
+            M.fail
+              (Errors.create
+                 ~context:
+                   "Failed accessing liquid balance from GraphQL \
```

Comment:
Can you change this copy to make it clear that the graphql request itself didn't fail, just that we weren't synced enough.

Suggestion:

"Unable to access liquid balance since your Mina daemon isn't fully bootstrapped"

Reply:
Sure, sounds good